plugins/source/aws/resources/services/ses/configuration_sets_fetch.go

```diff
@@ -12,27 +12,35 @@ func fetchSesConfigurationSets(ctx context.Context, meta schema.ClientMeta, pare
 	c := meta.(*client.Client)
 	svc := c.Services().Sesv2
 
-	config := sesv2.ListConfigurationSetsInput{}
-	p := sesv2.NewListConfigurationSetsPaginator(svc, &config)
+	p := sesv2.NewListConfigurationSetsPaginator(svc, nil)
 	for p.HasMorePages() {
 		response, err := p.NextPage(ctx)
 		if err != nil {
 			return err
 		}
 		res <- response.ConfigurationSets
 	}
+
 	return nil
 }
 
 func getConfigurationSet(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource) error {
 	c := meta.(*client.Client)
 	svc := c.Services().Sesv2
-	csName := resource.Item.(string)
+	csName := resource.GetItem().(string)
```

Comment:
Other than styling preferences, is there a reason for this change? If we prefer one way or the other maybe we could remove the public `Item`?

Reply:
No, only style (+ IDK if we'd want to protect the underlying Item in any way @yevgenypats )